portal/static/js/src/modules/TnthAjax.js

```diff
@@ -314,6 +314,24 @@ export default { /*global $ */
             }
         });
     },
+    getSubStudyTriggers: function(userId, params, callback) {
+        callback = callback || function() {};
+        if (!userId) {
+            callback({error: i18next.t("User id is required.")});
+            return false;
+        }
+        this.sendRequest(`/api/user/${userId}/triggers`, "GET", userId, params, function(data) {
```

Comment:
Great!  But I believe this is an implicit dependency on #3814?

Reply:
YES, but that is for the thank you modal, which needs more work (https://jira.movember.com/browse/TN-2644) UI wise. But that dependency shouldn't prevent the sub-study tiles from being rendered.